options/locale/locale_en-US.ini

```
@@ -1480,6 +1480,7 @@ issues.content_history.created = created
 issues.content_history.delete_from_history = Delete from history
 issues.content_history.delete_from_history_confirm = Delete from history?
 issues.content_history.options = Options
+issues.reference_link = References: %s
```

```suggestion
issues.reference_link = Reference: %s
```

ps: this spelling seems incorrect. It should be `Reference`.  Your screenshot of GitLab also uses `Reference`.

---

Thanks! I've (hopefully) corrected this in [f2aee4e](https://github.com/go-gitea/gitea/pull/19609/commits/f2aee4e565f2f074be1322a5ee20b468eb8584ec)